src/lambda/explorer/cloudFormationNodes.ts

```diff
@@ -16,29 +16,39 @@ import { AWSTreeErrorHandlerNode } from '../../shared/treeview/nodes/awsTreeErro
 import { AWSTreeNodeBase } from '../../shared/treeview/nodes/awsTreeNodeBase'
 import { ErrorNode } from '../../shared/treeview/nodes/errorNode'
 import { PlaceholderNode } from '../../shared/treeview/nodes/placeholderNode'
+import { makeChildrenNodes } from '../../shared/treeview/treeNodeUtilities'
 import { intersection, toArrayAsync, toMap, toMapAsync, updateInPlace } from '../../shared/utilities/collectionUtils'
 import { listCloudFormationStacks, listLambdaFunctions } from '../utils'
 import { LambdaFunctionNode } from './lambdaFunctionNode'
 
 export const CONTEXT_VALUE_CLOUDFORMATION_LAMBDA_FUNCTION = 'awsCloudFormationFunctionNode'
 
-export class CloudFormationNode extends AWSTreeErrorHandlerNode {
+export class CloudFormationNode extends AWSTreeNodeBase {
     private readonly stackNodes: Map<string, CloudFormationStackNode>
 
     public constructor(private readonly regionCode: string) {
         super('CloudFormation', vscode.TreeItemCollapsibleState.Collapsed)
         this.stackNodes = new Map<string, CloudFormationStackNode>()
     }
 
-    public async getChildren(): Promise<(CloudFormationStackNode | ErrorNode)[]> {
-        await this.handleErrorProneOperation(
-            async () => this.updateChildren(),
-            localize('AWS.explorerNode.cloudFormation.error', 'Error loading CloudFormation resources')
-        )
-
-        return !!this.errorNode
-            ? [this.errorNode]
-            : [...this.stackNodes.values()].sort((nodeA, nodeB) => nodeA.stackName.localeCompare(nodeB.stackName))
+    public async getChildren(): Promise<AWSTreeNodeBase[]> {
```

Comment:
@justinmk3  you saw a similar change in #814 , I'd be curious to hear your thoughts about whether or not this handling should get pushed up to `AWSTreeNodeBase`, and the hooks for the four methods would become overridable methods on the class instead.

Reply:
IMO it's fine as-is. `makeChildrenNodes()` already defines the "contract" so having another contract/hint in `AWSTreeNodeBase` doesn't gain much, AFAICT.